styles.css

```diff
@@ -17,6 +17,30 @@
     width: 100%;
   }
 
+  .mobile-menu {
+    width: 100%;
+    height: 100vh;
+    background-color: #3c3a39;
+    display: none;
+  }
+
+  .close-btn {
+    float: right;
+    padding-top: 20px;
+    padding-right: 35px;
+  }
+
+  .mobile-ul {
+    padding: 60px 20px;
+  }
+
+  .menu-li {
+    font-family: arial, sans-serif, 'inter';
+    font-size: 32px;
+    font-weight: 700;
+    padding: 10px 20px;
+  }
+
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=that%20you%20chose.-,You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins),-using%20the%20templates), Please add the correct margin top, bottom, and left values to the mobile menu options to make its position match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/223458694-8f10014f-da78-45d5-ba25-8c12bf60cb6d.png) | ![image](https://user-images.githubusercontent.com/68796232/223458672-7701364f-28e9-4546-80e6-43d1c59d7a27.png)  |